product_docs/docs/eprs/7/03_installation/01_installing_with_stackbuilder.mdx

```diff
@@ -1,195 +0,0 @@
----
```

Comment:
With this deleted, links to /eprs/latest/03_installation/01_installing_with_stackbuilder/ fail - which includes the links in /eprs/latest/05_smr_operation/01_prerequisites/06_verifying_host_accessibility/
/eprs/latest/05_smr_operation/02_creating_publication/01_registering_publication_server/
/eprs/latest/05_smr_operation/03_creating_subscription/01_registering_subscription_server/
/eprs/latest/10_appendix/02_upgrading_to_xdb6_2/02_upgrading_with_gui_installer/